client/flutter/lib/components/news_feed_item.dart

```diff
@@ -1,22 +1,28 @@
 import 'package:flutter/material.dart';
 import 'package:url_launcher/url_launcher.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
 
 class NewsFeedItem extends StatelessWidget {
   final String title;
   final String description;
   final ImageProvider imageProvider;
   final String url;
+  final FirebaseAnalytics analytics;
 
   NewsFeedItem(
       {@required this.title,
       @required this.description,
       @required this.imageProvider,
-      @required this.url});
+      @required this.url,
+      @required this.analytics});
 
   @override
   Widget build(BuildContext context) {
     return GestureDetector(
-      onTap: () => launch(this.url),
+      onTap: () {
+        analytics.logEvent(name: 'NewsFeedItem', parameters: {'url': url});
```

Comment:
Can we use `this.url` for consistency?
```suggestion
        analytics.logEvent(name: 'NewsFeedItem', parameters: {'url': this.url});
```